account-kit/signer/src/errors.ts

```diff
@@ -13,3 +13,9 @@ export class NotAuthenticatedError extends BaseError {
     );
   }
 }
+
+export class OAuthProvidersError extends BaseError {
```

Comment:
I see a lot of the other errors in this project having `override name = "ErrorName"`. Would be good to have that here I think (it looks like an oversight that it was missing from the other error in this file).

Reply:
Sounds good, added to both errors in this file.